vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_sku.cc

```diff
@@ -318,8 +318,8 @@ void ContributionSKU::Merchant(
       transaction,
       callback);
 
-  ledger_->GetSpendableUnblindedTokensByBatchTypes(
-      {ledger::CredsBatchType::PROMOTION},
+  ledger_->GetReservedUnblindedTokens(
```

Comment:
where are tokens reserved for this flow?

Reply:
Fixed now.